app/assets/stylesheets/base/_typography.scss

```diff
@@ -51,7 +51,8 @@ a:not([class]) {
 
   &:hover {
     animation: disco-color 2s forwards infinite;
-    border-width: 0;
+    border-color: rgba(0, 0, 0, 0);
```

Comment:
Curious about why this works!

Reply:
My limited understanding is that instead of removing the underline, this makes it transparent. I'll have to defer to Eric for a better explanation :)